resources/lang/en/Widget.properties

```
@@ -704,6 +706,14 @@ itemDataTableItemAgeRating = "Age rating"
 itemDataTableItemExternalId = "External variation ID"
 itemDataTableItemVariationModel = "Model"
 itemDataTableItemManufacturer = "Manufacturer"
+itemManufacturerLabel = "Item manufacturer"
```

Here, you added the German entries to the English properties file (and vice versa).

```suggestion
itemManufacturerLabel = "Item manufacturer"
itemManufacturerSampleName = "Name 1"
itemManufacturerSampleCompanyName = "Acme Company"
itemManufacturerSampleAddress = "Main Street 12"
itemManufacturerSampleLocation = "56009 Anytown Germany"
itemManufacturerSampleDomain = "info@sampledomain.com"
itemManufacturerSampleHomepage = "www.samplehomepage.com"
itemManufacturerSamplePhone = "+123456789"
```